src-electron/electron-main.ts

```diff
@@ -40,16 +44,21 @@ function createWindow() {
         width: windowSize.width,
         height: windowSize.height,
         useContentSize: true,
+        icon: path.resolve(__dirname, 'icons/icon.png'),
+        autoHideMenuBar: process.env.PROD,
         webPreferences: {
-            nodeIntegration: true,
-            nodeIntegrationInWorker: true,
-            webSecurity: false,
-            contextIsolation: false
-        },
-        icon: path.join(__dirname, 'icon.png'),
-        autoHideMenuBar: process.env.PROD
+            preload: path.resolve(
+                fileURLToPath(new URL('.', import.meta.url)),
+                path.join(process.env.QUASAR_ELECTRON_PRELOAD_FOLDER, 'electron-preload' + process.env.QUASAR_ELECTRON_PRELOAD_EXTENSION)
+            ),
+            // TODO - Remove and find an appropriate workaround for CORS blocking by Electron
```

Comment:
How confident are you in merging this PR before this TODO is resolved? Electron docs specifically recommend against doing this. I get the feeling that if we must rely on this, we're probably doing something wrong. Could it at least be limited to dev builds?

Reply:
Pre upgrade we had security disabled and so I'm not hugely concerned about this for now as part of the upgrade. Since it's an upgrade I don't think it matters too much as we're on-par with prior behaviour.

In theory we can flick the switch and things will work fine, we just need to resolve the CORS issue which we can do separately.